docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -82,7 +82,7 @@ In the webhook settings page, make sure that the webhook will only send Issue Up
 
 We recommend installing Teleport plugins alongside the Teleport Proxy. This is an ideal
 location as plugins have a low memory footprint, and will require both public internet access
-and Teleport Auth Service access. 
+and Teleport Auth Service access.
```

Comment:
Do plugins need direct access to the auth server or can they dial via proxy?

Reply:
They can do both. I haven't changed the docs here, but we could update this with mentioning the proxy as well

Reply:
I think it's okay to limit the changes to the Helm instructions in this PR—we'll be giving the access plugin guides a top-to-bottom edit in the next sprint or two.